.github/workflows/release.yml

```diff
@@ -33,8 +33,8 @@ jobs:
           gunzip -c "$destdir/$packagename.tar.gz" | xz > "$destdir/$packagename.tar.xz"
 
           (
-            cd "$destdir"
-            sha256sum * > .sha256
+            cd "$destdir" || exit
+            sha256sum ./* > .sha256
```

Comment:
Thank you for fix.
However, this will also cause the output to contain `./` in the output. So could you please use `--` instead?

  - ```console
    % sha256sum ./*.md
    953493f638b3ffc0c75d78f62479a28ffe352b2226e293232a7de5cff23edbde  ./CONTRIBUTING.md
    d22e92b7591ea18e67a9a4a1b35c912d87b6e87a2692cac8c66eac1bd22cac2e  ./README.md
    0021cdbff7ac726b3c29dfa2e768760996c00bfef6eb85f740cc16613c38181f  ./SECURITY.md
    4e25a5fe741e45abdda436d6b137cb312a1f848d49038125d1572d49020a8da2  ./TODO.md
    ```

  - ```console
    % sha256sum -- *.md
    953493f638b3ffc0c75d78f62479a28ffe352b2226e293232a7de5cff23edbde  CONTRIBUTING.md
    d22e92b7591ea18e67a9a4a1b35c912d87b6e87a2692cac8c66eac1bd22cac2e  README.md
    0021cdbff7ac726b3c29dfa2e768760996c00bfef6eb85f740cc16613c38181f  SECURITY.md
    4e25a5fe741e45abdda436d6b137cb312a1f848d49038125d1572d49020a8da2  TODO.md
    ```

Reply:
Thanks @dearblue all done now